util/buildRelease/install.sh

```diff
@@ -381,18 +381,23 @@ then
 fi
 
 CHPL_LANGUAGE_SERVER="bin/$CHPL_BIN_SUBDIR"/chpl-language-server
+# comes with chpl-language-serverh
+CHPL_SHIM="bin/$CHPL_BIN_SUBDIR"/chpl-shim
```

Comment:
You set this but don't check it. I think you need to duplicate the `if [ -f $CHPL_...` logic for `CHPL_SHIM`, to handle the case where they're separately built (i.e., not via `all` / `install`).

Reply:
In that case you don't need `CHPL_SHIM` as a variable, since you never use it (?).